web/themes/contrib/civictheme/includes/form.inc

```diff
@@ -100,7 +100,7 @@ function _civictheme_form_alter__non_form_elements__link(array &$element): void
  *
  * @SuppressWarnings(PHPMD.StaticAccess)
  */
-function _civictheme_form_preprocess_hidden_form_elements(array $variables): void {
+function _civictheme_form_preprocess_hidden_form_elements(array &$variables): void {
```

Comment:
This is needed, non exposed filters were not being added. 
`form_hidden_fields`
these be used when passed as query parameters.

Reply:
ok great thanks for checking

Reply:
So we should be deleting this function then.